expo/gbe/views/edit_act_techinfo_view.py

```diff
@@ -77,20 +78,30 @@ def EditActTechInfoView(request, act_id):
             except:
                 pass
             re_set.insert(0, existing_rehearsal)
+            re_set = sorted(re_set,
```

Comment:
If you're sorting the set, then the insert should be replaced by an append. 
Also, the sort should happen outside the loop.